### DISCUSSION TITLE
What is this "eight" (in my deployment link) ?

### DISCUSSION THREAD

**Discussion by alphandlouis** [vercel/community#3526] [Feature Question] [answered]
### Summary

Hello, I would like to know what is this "eight" (in my deployment link) ?

It was added in the link and I would like to know why because normally, there is nothing that is added like this in my deployments?

Link of deployment : https://react-officelite-coming-soon-site-eight.vercel.app/

The name of my site my repository is: react-officelite-coming-soon-site

Sincerely, Alphand Louis)

### Steps to Reproduce

It's just a question

**Comment by amyegan** [comment, marked as answer]
Hey @alphandlouis. When another project already has the same subdomain, a new project's subdomain will have a random extra word attached to make it unique. If you would like to customize the subdomain, you can add a new one in project settings. 

If you need a completely custom domain, it can also be added in project settings: https://vercel.com/docs/concepts/projects/domains/add-a-domain

**Reply by alphandlouis** [reply]
Thank You for your answer

[a comment by an automated account is not shown]
